app/controllers/v1/playlists_controller.rb

```diff
@@ -1,6 +1,6 @@
 module V1
   class PlaylistsController < ApplicationController
-    before_action :authenticate_request, only: [:create, :update, :destroy]
+    before_action :authenticate_request, only: [:update, :destroy]
```

Comment:
don't remove `create`
```suggestion
    before_action :authenticate_request, only: [:create, :update, :destroy]
```